registrace_bp.py

```diff
@@ -21,7 +21,7 @@ def add_new():
 
 	uz_registrovany = db_funkce.najdi_uzivatele(result.get("email"))
 	if uz_registrovany:
-		flash ('Zadaný e-mail už existuje v databázi. Můžeš se přihlásit.', "danger")
+		flash (Markup('Zadaný e-mail už existuje v databázi. <a href="/prihlaseni" class="alert-link"> Tady se přihlaš.</a>'), "danger")
```

Comment:
Co kdyby to místo odkazu na "Tady se přihlaš" to rovnou přesměrovalo na přihlašovací obrazovku, stějně jako tě to přesměruje když při přihlašování zadáš neexistující email, tak tě to hodí rovnou na registraci?

V každém případě po tom flashi chceš mít nějaký `return render_template(.....)`, aby ta funkce nepokračovala dál a nesnažila se toho člověka zaregistrovat nazdory tomu, že už víme, že tam ten účet existuje.